CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
-# Changelog
 
-All notable changes to this project will be documented in this file.
+FEATURES:
+* But now it shouldn't. ([GH-23])
+* Something else. ([GH-24])
 
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
-and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+IMPROVEMENTS:
+* Change project description to be even more exciting. ([GH-23])
 
-## [Unreleased]
-
-### Added
-
-- Add test binary and project skeleton.
-
-[Unreleased]: https://github.com/giantswarm/actions-test
+BUG FIXES:
+* Put changelogs in the correct directory. ([GH-24])
```

Comment:
@tfussell This looks incomplete. Where are the definitions of the references like `GH-23`?

Also I'd expect a headline per release, but I guess that has to come once a release is tagged, right?

Reply:
The current tool generates release notes suitable for a release PR description and the links work magically there (see this PR description). It will need a separate template for Keep-A-Changelog-style CHANGELOGs like we use, but that shouldn't be a problem.